src/CloudflareWorkers.fs

```diff
@@ -0,0 +1,380 @@
+// Generated with ts2fable 0.8.0 from https://github.com/Krzysztof-Cieslak/workers-types/blob/fable/index.d.ts
+module rec Fable.Cloudflare.Workers
+
+open Fable.Core
+open Fable.Core.JS
+open Fetch
+open Node.Stream
+
+type Array<'T> = System.Collections.Generic.IList<'T>
+
+type [<AllowNullLiteral>] IExports =
+    abstract HTMLRewriter: HTMLRewriterStatic
+
+type [<AllowNullLiteral>] FetchEvent =
+    /// The type of event.
+    abstract ``type``: string with get, set
+    /// The incoming HTTP request triggering `FetchEvent`.
+    abstract request: CfRequest with get, set
+    /// Extend the lifetime of the event without blocking the response
+    /// from being sent. Use this method to notify the runtime to wait
+    /// for tasks (e.g. logging, analytics to third-party services,
+    /// streaming and caching) that need to run longer than the usual
```

Comment:
I believe FetchEven is a type that should be part of the general Fable Workers API. I don't think it is specific CFWorkers. I looked about through the Fable Libraries once to see where it should go. I think it is the same thing as the type https://github.com/fable-compiler/fable-import/blob/30430945a92d6eb71e8644e056014528a534fa5e/Browser/Fable.Import.Browser.fs#L2849

No?

Reply:
Well, to be fair it probably should be something like `CfFetchEvent` that inherits from general `FetchEvent` - `waitUntil`, `passThroughOnException` etc are CF specific.

Just like `CfRequest` below inherits from general `Request`.

But I'm not sure if it's worth to complicate things. Unlike in case of `Request`, for `FetchEvent` in context of CF Workers you never should need to use any property that's not currently defined here in this type, so lack of inheritance and just modelling things as simple as possible is probably not an issue in practice.